### DISCUSSION TITLE
Set CPU/Memory limits on the Trivy scan-vulnerabilityreport pods

### DISCUSSION THREAD

**Discussion by jeremeb** [aquasecurity/starboard#571] [Questions & Help] [answered]
Hi All,

I am using the starboard-operator-0.5.1 chart. I have been able to set the CPU/Memory resource limits in the values file of the chart, but this does not affect the scan-vulnerabilityreport pods. They seem to always be at:

```
    Limits:
      cpu:     500m
      memory:  500M
    Requests:
      cpu:     100m
      memory:  100M
```

Is there a way to adjust these limits that I am not seeing? Or maybe this should be a feature request?

**Comment by danielpacak** [comment, marked as answer]
In v0.10.2 resource requests / limit for scan jobs are hardcoded. I opened #572 to make them configurable.

**Reply by jeremeb** [reply]
Thank you @danielpacak! If I can help at all, let me know.
